docs/basics/101-5-install.rst

```diff
@@ -0,0 +1,204 @@
+Basic DataLad magic: Installing datasets
+----------------------------------------
+
+So far, we have created a ``DataLad-101`` course dataset. We saved some additional readings
+into the dataset, and have carefully made and saved notes on the DataLad
+commands we discovered.
+
+But we've been told that with DataLad we could very easily get vast amounts of data to our
+computer. Rumor has it that this would be only a single command in the terminal!
+Therefore, everyone in todays lecture exitedly await todays topic: Installing datasets.
+
+"With DataLad, users can *install* existing
+DataLad datasets from paths, urls, or open-data collections." our lecturer begins.
+"This makes accessing data fast and easy. A dataset that others could install can be
+created by anyone, without a need for external software. Your own Datasets can be
+installed by others, should you want that, for example. Therefore, not only accessing
+data becomes fast and easy, but also *sharing*."
+
+"Thats so cool!", you think. "Exam preparation will be a piece of cake if all of us
+can share our mid-term and final projects easily!"
+
+"But today, lets only focus on how to install a dataset", she continuous.
+
+"Damn it! Can't we have longer lectures?", you think and set alarms to all of the
+upcoming lecture dates in your calender.
+There is so much exciting stuff to come, you can't miss a single one.
+
+Installing an existing dataset is done with the ``datalad install`` command.
+The command takes a location of an existing dataset (the *source*), and a path to where you want
+the dataset to be installed. The source can be a URL or a path to a local directory,
+or an SSH server. Additionally, it can also be a pointer to an open-data collection,
+for example :term:`the DataLad superdataset ///` (more on this later, though).
+
+Let's try this. To practice, we're using one of the many openly available datasets that
+DataLad provides in a public registry that anyone can access. One of these datasets is a
+collection of great (and free) machine learning books. The DataLad-101 course is not
+a machine-learning course, but those are nevertheless a good addition to the large stash
+of yet-to-read text books we piled up. Lets install this dataset into our exisiting
+DataLad-101 dataset, inside the directory books.
+
+Because we are installing a dataset within an existing dataset, we supply the ``-d``
+(``--dataset``) flag. This specifies the dataset to perform the operation on. Because we are inside
+of the ``DataLad-101`` dataset, the pointer to the dataset is a ``.`` (which is Unix
+way for saying "current directory"). The dataset to be installed lives on Github, and
+we can give its Github URL as a source (``-s``, ``--source``).
+
+.. runrecord:: _examples/DL-101-5-1
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $  datalad install -d . -s https://github.com/datalad-datasets/machinelearning-books.git books/ml-books
```

Comment:
I am torn re the use of short options. They keep things at a comprehensible length, but they are also harder to parse for people that see them for the first time. Maybe it is better to introduce them like this:
```
datalad install --dataset . --source https://github.com/datalad-datasets/machinelearning-books.git books/ml-books

# or shorter
#datalad install -d . -s https://github.com/datalad-datasets/machinelearning-books.git books/ml-books
```
in the same box. With the comments it may be clear enough that only one of the lines needs to be executed.